queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```

Comment:
- [ ] It would be better if we make the right query on `line 4` as we should list the `name` of all animals born between 2016 and 2019 as seen in the project requirements
![Screenshot from 2023-05-02 13-11-50](https://user-images.githubusercontent.com/46388113/235675257-57da3b5c-46e9-4410-b81b-20695d6ca2a2.png)
The wildcard `*` will list all columns and that is not what we want to display